lib/scope.js

```diff
@@ -109,8 +111,8 @@ module.exports = function(options) {
         cb();
       } else {
         debug('Insufficient scope: ', tokenScopes);
-        cb(new oauth2Provider.TokenError(
-          'Insufficient scope', 'insufficient_scope', null, 403));
+        cb(new oauth2Provider.TokenError(g.f(
+          'Insufficient scope', null, 403), 'insufficient_scope'));
```

Comment:
I think it should be 
`oauth2Provider.TokenError(g.f('Insufficient scope'), 'insufficient_scope', null, 403)`?